modules/liveIntentAnalyticsAdapter.js

```diff
@@ -1,134 +1,118 @@
+    liip: encodeBoolean(liveIntentIdsPresent) // whether or not the LiveIntent IDs are present in one of the bid requests of the auction
+  };
+  const filteredData = ignoreUndefined(data);
+  sendData('bid-won', filteredData);
 }
 
-function createAnalyticsEvent(args, winningBids) {
-  let payload = {
-    instanceId: generateUUID(),
-    url: getRefererInfo().page,
-    bidsReceived: getAnalyticsEventBids(args.bidsReceived),
-    auctionStart: args.timestamp,
-    auctionEnd: args.auctionEnd,
-    adUnits: [],
-    userIds: [],
-    bidders: []
-  }
-  let allUserIds = [];
-
-  if (args.adUnits) {
-    args.adUnits.forEach(unit => {
-      if (unit.mediaTypes && unit.mediaTypes.banner) {
-        payload['adUnits'].push({
-          code: unit.code,
-          mediaType: 'banner',
-          sizes: getBannerSizes(unit.mediaTypes.banner),
-          ortb2Imp: unit.ortb2Imp
-        });
-      }
-      if (unit.bids) {
-        let userIds = unit.bids.flatMap(getAnalyticsEventUserIds);
-        allUserIds.push(...userIds);
-        let bidders = unit.bids.map(({bidder, params}) => {
-          return { bidder, params }
-        });
-
-        payload['bidders'].push(...bidders);
-      }
-    })
-    let uniqueUserIds = getUniqueBy(allUserIds, 'source');
-    payload['userIds'] = uniqueUserIds;
-  }
-  payload['winningBids'] = getAnalyticsEventBids(winningBids);
-  payload['auctionId'] = args.auctionId;
-  return payload;
+function encodeBoolean(value) {
+  return value === undefined ? undefined : value ? 'y' : 'n'
 }
 
-function getAnalyticsEventUserIds(bid) {
-  if (bid && bid.userIdAsEids) {
-    return bid.userIdAsEids.map(({source, uids, ext}) => {
-      let analyticsEventUserId = {source, uids, ext};
-      return ignoreUndefined(analyticsEventUserId)
-    });
-  } else { return []; }
+function checkLiveIntentIdsPresent(bidRequests) {
+  const eids = bidRequests?.flatMap(r => r?.bids).flatMap(b => b?.userIdAsEids);
+  return !!eids.find(eid => eid?.source === 'liveintent.com') || !!eids.flatMap(e => e?.uids).find(u => u?.ext?.provider === 'liveintent.com')
 }
 
-function sendAnalyticsEvent(data) {
-  ajax(URL, {
-    success: function () {
-      logInfo('LiveIntent Prebid Analytics: send data success');
-    },
-    error: function (e) {
-      logWarn('LiveIntent Prebid Analytics: send data error' + e);
-    }
-  }, JSON.stringify(data), {
-    contentType: 'application/json',
-    method: 'POST'
-  })
+function sendData(path, data) {
+  const fields = Object.entries(data);
+  if (fields.length > 0) {
+    const params = fields.map(([key, value]) => key + '=' + encodeURIComponent(value)).join('&');
+    ajax(URL + '/' + path + '?' + params, undefined, null, { method: 'GET' });
+  }
 }
 
 function ignoreUndefined(data) {
-  const filteredData = Object.entries(data).filter(([key, value]) => value)
-  return Object.fromEntries(filteredData)
+  const filteredData = Object.entries(data).filter(([key, value]) => value);
```

Comment:
Can `Object.entries(data)` be undefined with the new setting?

Reply:
As far as I can see, there is not way for it to be undefined.